build/cwl-wrappers/cmo-marianas/0.0.0/cmo-marianas.cwl

```diff
@@ -0,0 +1,102 @@
+cwlVersion: cwl:v1.0
+
+#!/usr/bin/env/cwl-runner
+
+$namespaces:
+  dct: http://purl.org/dc/terms/
+  foaf: http://xmlns.com/foaf/0.1/
+  doap: http://usefulinc.com/ns/doap#
+
+$schemas:
+- http://dublincore.org/2012/06/14/dcterms.rdf
+- http://xmlns.com/foaf/spec/20140114.rdf
+- http://usefulinc.com/ns/doap#
+
+doap:release:
+- class: doap:Version
+  doap:name: innovation-umi-trimming
+  doap:revision: 0.5.0
+- class: doap:Version
+  doap:name: cwl-wrapper
+  doap:revision: 1.0.0
+
+dct:creator:
+- class: foaf:Organization
+  foaf:name: Memorial Sloan Kettering Cancer Center
+  foaf:member:
+  - class: foaf:Person
+    foaf:name: Ian Johnson
+    foaf:mbox: mailto:johnsoni@mskcc.org
+
+dct:contributor:
+- class: foaf:Organization
+  foaf:name: Memorial Sloan Kettering Cancer Center
+  foaf:member:
+  - class: foaf:Person
+    foaf:name: Ian Johnson
+    foaf:mbox: mailto:johnsoni@mskcc.org
+
+class: CommandLineTool
+
+baseCommand: [cmo_process_loop_umi_fastq]
+
+arguments: ["-server", "-Xms8g", "-Xmx8g", "-cp"]
```

Comment:
I had to modify `sing-java.sh` to accomodate the use of `-cp` instead of `-jar`, not sure if its the best solution but that's the way this jar is built (no default class is specified).